src/test/java/com/epam/tc/hw1/TestDiv.java

```diff
@@ -0,0 +1,50 @@
+package com.epam.tc.hw1;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import com.epam.tat.module4.Calculator;
+import org.assertj.core.data.Percentage;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+
```

Comment:
Let's name our test classes DivTest and so on, not TestDiv. Please, do the same for other classes.